.aegir.js

```diff
@@ -4,6 +4,7 @@ const multiaddr = require('multiaddr')
 const pull = require('pull-stream')
 const WS = require('libp2p-websockets')
 const PeerId = require('peer-id')
+const prom = (fnc) => new Promise((resolve, reject) => fnc((err, res) => err ? reject(err) : resolve(res)))
```

Comment:
This turns callbacks into promises

Reply:
Why not use https://www.npmjs.com/package/promisify ? (I mean, if this one while works perfectly for all cases, then great)

Reply:
It covers most of the use cases and doesn't require an extra module. Also it has a more convenient syntax: Instead of having to first promisify a module and then use it, I can simply do `prom(cb => anything(anyArgument, cb))` and await that.

I could toss it into a module, but I found that overkill since it's pretty easy to remember after using it for a while